src/cloudWatchLogs/commands/addLogEvents.ts

```diff
@@ -47,4 +47,5 @@ export async function addLogEvents(
     if (onDidChangeCodeLensEvent) {
         onDidChangeCodeLensEvent.fire()
     }
+    registry.highlightDocument(uri)
```

Comment:
If you start adding the same function call to change UI state in a bunch of different places, that probably means something is off with the control flow and/or data flow of the overall design. I wouldn't change anything though as this is minor (for now).